src/utils/equipment.ts

```diff
@@ -13,7 +13,8 @@ export function queryEquipment(
           equipment.description
             .toLowerCase()
             .includes(query.toLowerCase().trim()) ||
-          equipment.category
+          equipment.category.path ||
```

Comment:
We'll have to figure out what's happening with the `path || name` noise that appears in this file as well.
Should equipment just have a path?  The idea of spitting it on the category is to deal with the hierarchy.  When we apply the category to the idea, we could just apply `path + name` as the category string.

I'm also still seeing what I think are arbitrary choices of when to `.toLowerCase().trim()` some string.
There should be a clear pipeline for all these string comparisons.  The `.toLowerCase().item()` should happen as the string enters the pipeline, not before.

Moreover, in `filterEquipment` for instance, the comparison is coming from the same data source, no?  I understand needing to process user input, but if the tags being compared are from the same data source, their casing and white space should be already identical... or am I missing something?

Either way, I think these deep utilities should assume that the caller as already cleaned up the input.

Reply:
I think either a getter `item.topCategory` would be great, I've still yet to change the EquipmentForm to acknowledge the existence of Sub Categories, so perhaps once I've done that I'll adjust distribution of items by category accordingly